generator.py

```diff
@@ -11,32 +10,29 @@
 NEED_UPDATE = 1
 FAILURE = 2
 
-DEFAULT_GIR_DIRECTORY = 'gir-files'
-DEFAULT_GIR_PATH = './gir/target/release/gir'
+DEFAULT_GIR_FILES = Path('./gir-files')
+DEFAULT_GIR_DIRECTORY = Path('./gir/')
+DEFAULT_GIR_PATH = DEFAULT_GIR_DIRECTORY / 'target/release/gir'
 
 
 def run_command(command, folder=None):
     if folder is None:
         folder = "."
-    child = subprocess.Popen(command, cwd=folder)
-    child.communicate()
-    if child.returncode != 0:
-        print("Command `{}` failed with return code `{}`...".format(command, child.returncode))
+    ret = subprocess.run(command, cwd=folder)
+    if ret.returncode != 0:
+        print("Command `{}` failed with `{}`...".format(command, ret))
         return False
     return True
 
 
 def update_workspace():
-    try:
-        return run_command(['cargo', 'build', '--release'], 'gir')
-    except:
-        return False
+    return run_command(['cargo', 'build', '--release'], 'gir')
 
 
 def ask_yes_no_question(question, conf):
     question = '{} [y/N] '.format(question)
     if conf.yes:
-        print(question)
+        print(question + 'y')
```

Comment:
Why adding the `y` there?

Reply:
I'd rather not print this at all. It's a bit weird to see `[y/N]` (fullcaps `N` meaning default no show up) but the program continuing, with seemingly no answer given.

Reply:
Well, I guess it makes sense in a way. Let's go for that then!